src/views/ThreeWActivityForm/ActivitiesBySectorInput/ActivityInput/ActionSupplyInput/index.tsx

```diff
@@ -0,0 +1,82 @@
+import {
+    ArrayError,
+    useFormObject,
+    SetValueArg,
+    getErrorObject,
+} from '@togglecorp/toggle-form';
```

Comment:
When importing types let's indicate that they are types
```suggestion
import {
    type ArrayError,
    useFormObject,
    type SetValueArg,
    getErrorObject,
} from '@togglecorp/toggle-form';
```